solana/utils/validate.py

```diff
@@ -0,0 +1,29 @@
+"""Validation utilities."""
+from __future__ import annotations
+
+from enum import IntEnum
+from typing import Any
+
+from solana.transaction import TransactionInstruction
+
+
+def validate_instruction_keys(instruction: TransactionInstruction, expected: int) -> None:
```

Comment:
`validate_instruction_keys_len` sounds clearer to me.

Reply:
It's just a small thing but will be nicer if you at least respond to this before merging. 🤔 
Also, the renaming turns out breaking the downstream `pyserum`, maybe we should fix the version of the `solana-py` that we're using and upgrade the version in PR.